operator_ui/src/Layout.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import universal from 'react-universal-component'
+import {
+  Route,
+  Switch,
+  Redirect,
+  BrowserRouter as Router,
+} from 'react-router-dom'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import Private from './Private'
+import Loading from 'components/Loading'
+import { useOperatorUiSelector } from 'reducers'
+
+// Asynchronously load routes that are chunked via code-splitting
+// 'import' as a function must take a string. It can't take a variable.
+const uniOpts = { loading: Loading }
+const SignIn = universal(import('./pages/SignIn'), uniOpts)
+
+const Layout = () => {
+  // Remove the server-side injected CSS.
+  const jssStyles = document.getElementById('jss-server-side')
```

Comment:
Do you know why we are removing SS injected CSS?

Reply:
🤷‍♂️ didn't investigate.

Reply:
I assume because the styles are re-generated on FE, which is not clean as they should simply be taken over, but 🤷‍♂️